core/src/components/input/input.ionic.scss

```diff
@@ -1,26 +1,127 @@
 @use "../../themes/ionic/ionic.globals.scss" as globals;
 @use "./input.common";
-@forward "./input.ionic.outline.scss";
-
+@forward "./input.ionic.solid.scss";
 // Ionic Input
 // --------------------------------------------------
 
 :host {
   --color: #{globals.$ion-primitives-neutral-1200};
   --border-width: #{globals.$ion-border-size-025};
```

Comment:
We should move all styles specific to `fill` to the `input-fill-outline` class like we do for button: 

- [button-solid](https://github.com/ionic-team/ionic-framework/blob/1877ac5dd13ddc0a969c75147d08558b3093f117/core/src/components/button/button.ionic.scss#L45)
- [button-outline](https://github.com/ionic-team/ionic-framework/blob/1877ac5dd13ddc0a969c75147d08558b3093f117/core/src/components/button/button.ionic.scss#L59)

This will make it so that in the future if we need to change the default fill we don't have to move the styles around.